fission-web-server/sql/README.md

```diff
@@ -4,11 +4,22 @@
 This folder contains SQL files that should be run to get our database up to date. Each is prefixed with the unix time (in seconds) when it was created.
 
 ### Running the files
-You can run them using `\i` in you postgres terminal.
+
+You can run them using `\i` in your postgres terminal.
 
 ```sql
 \i 1576714308_selda_to_persistent.sql
 ```
 
+Note that `\i` assumes a relative path.
+
+If you are running postgres in a Docker container, connect to container locally then run the migration file
+
+```sh
+psql -U postgres -d web_api -h runfission.test -p 5432
```

Comment:
We should (could?) add `postgresql` to the nix shell environment for convenience - wanna do it in this PR?

Reply:
Ah yeah, I see the comment above. We'll get this in and update the shell separately. 👌